packages/preview/letterloom/0.1.0/README.md

```diff
@@ -0,0 +1,229 @@
+# The `letterloom` Package
+<!-- markdownlint-disable MD033 -->
+<div align="center">Version 0.1.0</div>
+
+The `letterloom` package is a user-friendly and customizable template designed to streamline the creation of professional-looking letters.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on the Typst web app.
+
+```typ
+#import "@preview/letterloom:0.1.0": *
+
+#show: letterloom.with(
+  // Sender's contact information (name and address)
+  from: (
+    name: "Sender's Name",
+    address: [Sender's Address]
+  ),
+
+  // Recipient's contact information (name and address)
+  to: (
+    name: "Receiver's Name",
+    address: [Receiver's Address]
+  ),
+
+  // Letter date (automatically set to today's date)
+  date: datetime.today().display("[day padding:zero] [month repr:long] [year repr:full]"),
+
+  // Opening greeting (e.g., "Dear Mr Hawthorne,")
+  salutation: "Dear Receiver's Name,",
+
+  // Letter subject line
+  subject: "Subject",
+
+  // Closing phrase (e.g., "Yours sincerely,")
+  closing: "Yours sincerely,",
+
+  // List of signatories with their names
+  signatures: (
+    (
+      name: "Sender's Name",
+      // signature: image() // Add your signature image here
+    )
+  ),
+)
+
+// Write the body of your letter here
+```
+
+### Installation
```

Comment:
I'm not sure this section and the ones below are relevant for Typst Universe users. If you want to keep them, I think it would be nice to make it more explicit that they are intended for contributors and developers and not users.

Reply:
Understood, but what should the Installation section have? I can add an extra section called Development and move all this information into that section.